hack/test-pj.sh

```diff
@@ -1,23 +1,38 @@
 #!/usr/bin/env bash
 # Inspired by https://github.com/kubernetes/test-infra/blob/047b0a0596f7d42b909405c0b8ed4366de2bef72/prow/pj-on-kind.sh
 # Required kubectl
-# Rightnow this work only on the x86 platform(Linux, MacOS)
+# Right now this work only on the x86 platform(Linux, MacOS)
 
 set -o errexit
-set -o nounset
 set -o pipefail
 
 
 function main() {
+  if [[ "$1" == "-h" || "$1" == "--h" || "$1" == "--help" ]];then
+    echo -e "USAGE:\n testpj job branch [purpose]"
+    exit 0
+  fi
+  if [[ "$#" -lt 2 ]];then
+    echo -e "USAGE:\n testpj job branch [purpose]"
+    exit 1
```

Comment:
This is making the script pass more than one argument as mandatory. IMO it should still allow users to pass only the job name and go ahead.
This act of adding purpose to a local file should be optional if a user wishes to.

I suggest you have an optional flag maybe with the name "-purpose" which on enabling True would create/concatenate details to a local file.